src/Radicle/Internal/Effects.hs

```diff
@@ -82,7 +82,7 @@ replPrimFns = fromList $ allDocs $
       , "Pretty-prints a value."
       , \case
         [x] -> do
-            putStrS (renderPrettyDef x)
+            putStrS (renderAnsi x)
```

Comment:
I think this should stay as `renderPrettyDef`, since otherwise we can't read it back!

Reply:
We might add another primop for this, or options to this one, later.